task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -391,10 +430,53 @@ def run(self, task_config: KubernetesTaskConfig) -> Optional[str]:
         return None
 
     def reconcile(self, task_config: KubernetesTaskConfig) -> None:
-        # TASKPROC(244): we probably only want reconcile() to fill in the task_config
-        # member of the relevant KubernetesTaskMetadata and do an unconditional state
-        # reconciliation on startup
-        pass
+        pod_name = task_config.pod_name
+        try:
+            pod = self.kube_client.get_pod(namespace=self.namespace, pod_name=pod_name)
+        except Exception:
+            logger.exception(f"Cannot reconcile pod {pod_name}")
```

Comment:
should we set `pod = None` here so that we correctly enter line 452?

Reply:
This was originally just `return` ing here but lost the line during some updates, which would have left the action in a perpetual unknown state.  

I think it seems more user-friendly to treat this as LOST, so I'll go your `pod = None` route.